core/src/main/java/org/apache/iceberg/view/BaseMetastoreViewCatalog.java

```diff
@@ -79,6 +79,7 @@ protected BaseViewBuilder(TableIdentifier identifier) {
       Preconditions.checkArgument(
           isValidIdentifier(identifier), "Invalid view identifier: %s", identifier);
       this.identifier = identifier;
+      this.properties.putAll(tableDefaultProperties());
```

Comment:
does it make sense to pass properties prefixed with `table-default.` to views? I think passing properties prefixed with `view-default.` seems more reasonable to me, which is being handled in https://github.com/apache/iceberg/pull/11064